docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -136,6 +136,24 @@ General-purpose aggregation functions
 
         Examples are the same as for :attr:`ArrayAgg.ordering`.
 
+    Usage example::
+
+        class Person(models.Model):
+            info = models.JSONField()
+
+        >>> from django.contrib.postgres.aggregates import JSONBAgg
+        >>> Person.objects.create(info={'name': 'foo'})
+        >>> Person.objects.aggregate(jsonbagg=JSONAgg('info'))
+        {'jsonbagg': [{'name': 'mariyam'}]}
+
+    When used to aggregate on a :ref:`JSONField <django.db.models.JSONField>`,
+    key or index based transformations can be chained with it.
```

Comment:
Maybe worth calling back to the relevant docs section for more examples? (ie: [this bit on "Key, index, and path transforms"](https://docs.djangoproject.com/en/dev/topics/db/queries/#key-index-and-path-transforms) or something? There might be a better docs link that I'm not aware of...)

Reply:
Seems legit. For the time being, I have updated this text to link to your proposed example till someone gives a potentially better link.

Thanks.